test/tests/components/database/db_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package database
+
+import (
+	"context"
+	"testing"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/gitpod-io/gitpod/test/pkg/integration"
+	test_context "github.com/gitpod-io/gitpod/test/pkg/integration/context"
+)
+
+func TestBuiltinUserExists(t *testing.T) {
+	builtinUser := features.New("database").
+		WithLabel("component", "database").
+		Setup(func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			api := integration.NewComponentAPI(ctx, cfg.Namespace(), cfg.Client())
+			return test_context.SetComponentAPI(ctx, api)
+		}).
+		Assess("it should exists a builtin user workspace", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
```

Comment:
... as does this description. It is in multiple places

Reply:
fixed